addons/music/fnc_isMusicPlaying.sqf

```diff
@@ -20,4 +20,19 @@ Author:
     Dedmen, Commy2, Fishy
 ---------------------------------------------------------------------------- */
 
-!isNil QGVAR(track)
+#define MINIMUM_WAIT 0.1
```

Comment:
Explain.

Reply:
In order to check if `playMusic ""` was used to stop the music, the function checks if the current mission time and the mission time when the last track was started differ. If they do differ check if `getMusicPlayedTime` equals 0 if so that would mean that there is currently no music playing. However `getMusicPlayedTime` seems to be an estimation and can result in the same value over multiple frames, leading to false positives if the track just started. In order to prevent these false positives the function doesn't check if `getMusicPlayedTime` equals 0 if the current time and the starting time are withing `MINIMUM_WAIT` of each other.

Reply:
So if getMusicPlayedTime returns garbo, how do we know 0.1s is enough?

Reply:
I have now done more in depth testing on both my desktop and laptop, in a locally hosted server with only one player. ([script I used for testing](https://pastebin.com/qG6qbaWB))

I tested a variety of FPSes taking 150 samples for each, these are my results:

FPS | Max Time | Average Time
--- | --- | ---
2 | 2.333010 | 1.155892
10 | 0.385986 | 0.300724
20 | 0.240967 | 0.191904
36 | 0.170898 | 0.157701
59 | 0.150009 | 0.126583
120 | 0.120972 | 0.106157
454 | 0.100006 | 0.090674

Based on this its safe to say 0.1s is definitely not enough.

This leaves me with a question: what would you like me to do?
Count for worst case scenario? Would have to test 1 FPS then.
Have a minimum delay of 2.5s, and switch that  to 0.5s if they have 10+ fps?
Or go even further and base the delay on a FPS - Minimum delay table?

Keep in mind that this only to account for `playMusic ""`, CBA_fnc_stopMusic works perfectly fine.

Reply:
I have gone with the FPS based switch since it pretty simple and still responsive.